### PR TITLE
Allow to use centered title in AdaptiveTopAppBar with Material3 theme using MaterialTopAppBarAdaptation

### PR DESCRIPTION
**Pull request by overpas**
Screenshot_20240415_154435

### CODE REVIEW

File: cupertino-adaptive/src/commonMain/kotlin/io/github/alexzhirkevich/cupertino/adaptive/AdaptiveTopAppBar.kt [hunk 1]
```diff
@@ -67,23 +68,58 @@ fun AdaptiveTopAppBar(
...
+}
+
 @Stable
 @OptIn(ExperimentalMaterial3Api::class)
 class MaterialTopAppBarAdaptation internal constructor(
-    colors : TopAppBarColors
+    colors: TopAppBarColors,
+    centeredTitle: Boolean = false,
 ) {
+    var centeredTitle: Boolean by mutableStateOf(centeredTitle)
```
Comment (alexzhirkevich) [inline cupertino-adaptive/src/commonMain/kotlin/io/github/alexzhirkevich/cupertino/adaptive/AdaptiveTopAppBar.kt:122]: It's better to use `isCenterAligned` instead

Comment (overpas) [reply cupertino-adaptive/src/commonMain/kotlin/io/github/alexzhirkevich/cupertino/adaptive/AdaptiveTopAppBar.kt]: Fixed

File: cupertino-adaptive/src/commonMain/kotlin/io/github/alexzhirkevich/cupertino/adaptive/AdaptiveTopAppBar.kt [hunk 2]
```diff
@@ -67,23 +68,58 @@ fun AdaptiveTopAppBar(
...
+}
+
 @Stable
 @OptIn(ExperimentalMaterial3Api::class)
 class MaterialTopAppBarAdaptation internal constructor(
-    colors : TopAppBarColors
+    colors: TopAppBarColors,
+    isCenterAligned: Boolean = false,
 ) {
+    var isCenteredAligned: Boolean by mutableStateOf(isCenterAligned)
```
Comment (alexzhirkevich) [inline cupertino-adaptive/src/commonMain/kotlin/io/github/alexzhirkevich/cupertino/adaptive/AdaptiveTopAppBar.kt:122]:
```suggestion
    var isCenterAligned: Boolean by mutableStateOf(isCenterAligned)
```

File: cupertino-adaptive/src/commonMain/kotlin/io/github/alexzhirkevich/cupertino/adaptive/AdaptiveTopAppBar.kt [hunk 3]
```diff
@@ -67,23 +68,58 @@ fun AdaptiveTopAppBar(
             )
         },
         material = {
-            TopAppBar(
+            SingleRowTopAppBar(
                 title = title,
+                isCenterAligned = it.isCenteredAligned,
```
Comment (alexzhirkevich) [inline cupertino-adaptive/src/commonMain/kotlin/io/github/alexzhirkevich/cupertino/adaptive/AdaptiveTopAppBar.kt:73]:
```suggestion
                isCenterAligned = it.isCenterAligned,
```

**Review by alexzhirkevich — APPROVED** [review]

**Pull request merged**